server/src/main/java/org/elasticsearch/index/engine/ThreadPoolMergeScheduler.java

```diff
@@ -50,7 +50,7 @@ public class ThreadPoolMergeScheduler extends MergeScheduler implements Elastics
     );
     private final ShardId shardId;
     private final MergeSchedulerConfig config;
-    private final Logger logger;
+    protected final Logger logger;
```

Comment:
Unrelated change, in order to let these log messages go to the `ThreadPoolMergeScheduler`'s logger instead of the `InternalEngine`'s.